Formula/o/odin.rb

```diff
@@ -18,18 +18,112 @@ class Odin < Formula
     sha256 cellar: :any_skip_relocation, x86_64_linux:   "43f62fc9cca1fad774659c6da3ce6a9ea23304318b88970e2509386b30e0d436"
   end
 
+  depends_on "cmake" => :build
   depends_on "llvm"
 
+  on_macos do
+    resource "glfw" do
+      url "https://github.com/glfw/glfw/archive/refs/tags/3.3.8.tar.gz"
+      sha256 "f30f42e05f11e5fc62483e513b0488d5bceeab7d9c5da0ffe2252ad81816c713"
+    end
+
+    resource "raylib" do
+      url "https://github.com/raysan5/raylib/archive/refs/tags/5.0.tar.gz"
+      sha256 "98f049b9ea2a9c40a14e4e543eeea1a7ec3090ebdcd329c4ca2cf98bc9793482"
+    end
```

Comment:
Homebrew provides formulae for both of these. Can they be used instead?

Reply:
I can give it a try for raylib, I didn't because there seem to be weirdness with it on homebrew, `brew install raylib` installs `libraylib.4.5.0.dylib` even though it says it is raylib 5.

For glfw brew just installs a dynamic library, not a static one, so that won't work in its current state.

Reply:
> I can give it a try for raylib

Please.

> For glfw brew just installs a dynamic library, not a static one, so that won't work in its current state.

We'd welcome a PR adding a static library to that formula.

Reply:
> I didn't because there seem to be weirdness with it on homebrew, `brew install raylib` installs `libraylib.4.5.0.dylib` even though it says it is raylib 5.

That's due to upstream doing release without version update and there being no release after fix went in https://github.com/raysan5/raylib/commit/032cc497ca5aaca862dc926a93c2a45ed8017737